tests/test_tfkeras.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+import optuna
+from optuna._imports import try_import
```

Comment:
Ditto.

Reply:
Fixed in https://github.com/optuna/optuna-integration/pull/21/commits/1dd2ebdb8aa183558b66c0f732381fe501e7c2d7.